.launches/Run failed tests.launch

```diff
@@ -7,7 +7,7 @@
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
 <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="org.testng.remote.RemoteTestNG"/>
 <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="teammates"/>
-<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-DfailedTests=&quot;testStudentAccessControlInOrder,classSetup,testHelpLink,testStudentLogin,testInstructorAccessControlInOrder,testInstructorCourseAddPage,testInstructorHomePage,testDataBundle,testSetup,testPersistenceAndDeletion,testInstructorEvalPage&quot;  -D-Xss2m"/>
+<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-DfailedTests=&quot;testStudentAccessControlInOrder,classSetup,testHelpLink,testStudentLogin,testInstructorAccessControlInOrder,testInstructorCourseAddPage,testInstructorHomePage,testDataBundle,testSetup,testPersistenceAndDeletion,testInstructorEvalPage&quot;  -D-Xss2m -Duser.timezone=UTC"/>
```

Comment:
It is a change on my local computer, shouldn't have pushed it. Is there any way to revert it?